lib/iris/tests/unit/fileformats/pp/test_save.py

```diff
@@ -324,10 +324,14 @@ def _get_single_time_cube(set_time_mean=False):
     cube.remove_coord("time")
     cube.remove_coord("forecast_period")
     tc = DimCoord(
-        points=[15,],
+        points=[
+            15,
+        ],
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?